lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -24,6 +24,8 @@ class WorldpayGateway < Gateway
         'elo'              => 'ELO-SSL'
       }
 
+      UNKNOWN_CARD_CODE = 'CARD-SSL'
```

Comment:
I think it might be more consistent with the existing code to place `CARD-SSL` into the map above. Perhaps with the key "unknown".

Reply:
I actually tried that first, but the problem is the key it looks for isn't `unknown`. For example, if we pass in a sodexo card that's not supported / unknown for worldpay `credit_card('6060704495764400', brand: 'sodexo')`, the key here would be `sodexo`. Let me know if I'm missing something...